apps/client/src/features/cuesheet/cuesheet-table-settings/CuesheetTableSettings.tsx

```diff
@@ -38,6 +40,9 @@ function CuesheetTableSettings(props: CuesheetTableSettingsProps) {
       <div className={style.leftPanel}>
         <div className={style.sectionTitle}>Toggle column visibility</div>
         <div className={style.options}>
+          <label className={style.option}>
+            <Checkbox variant='ontime-ondark' defaultChecked={showIndexColumn} onChange={() => toggleIndexColumn()} />#
```

Comment:
suggestion: maybe we call this Index instead of `#`? what do you think?

Reply:
I didn't mind, of course. I chose that name because the column header is named `#`, so inconsistent naming may be confusing for users

Reply:
I agree. The # convention is common as an index in other apps, we chose it here as a way to keep the text small since we want the column to take as least space as possible.

I wonder whether having a name here would clarify the dataset for users. Potentially we can do `Index #`

I am not too sure either way so will leave up to your judgement

Reply:
> Index #

That's a good compromise :) 

Pushing a commit

Reply:
Resolving with https://github.com/cpvalente/ontime/pull/1037/commits/4961949ea0b0cb1a5fe2e3afd50eb9148a61399c